cmd/init.js

```diff
@@ -1,16 +1,35 @@
-const fsExstra = require('fs-extra');
+const fsExtra = require('fs-extra');
 const path = require('path');
+const fs = require("fs");
+
 
 const {
     getPackageRoot
 } = require('./lib/packageInfo');
 
 async function initCmd(argv) {
 
-    // check for project	
-    await fsExstra.copy(
-        path.join(getPackageRoot(), "template"), '.'
+    let directoryToCopyTo = argv.dir;
+    if (!checkIfDirectoryIsValidAndWritable(directoryToCopyTo)) {
+        console.error(`Directory ${directoryToCopyTo} is not valid!`);
```

Comment:
Try to create the directory if it doesn't exist. If you don't have rights throw an error.